sonar-vbnet-plugin/src/main/java/org/sonar/plugins/vbnet/VbNetPlugin.java

```diff
@@ -79,7 +79,6 @@ public void define(Context context) {
       GeneratedFileFilter.class,
       WrongEncodingFileFilter.class,
       // importers / exporters
-      AnalysisWarningsSensor.class,
```

Comment:
If we register the sensor twice we cannot avoid loading the warnings multiple times.